tests/sentry/integrations/jira_server/test_installation.py

```diff
@@ -0,0 +1,433 @@
+import responses
```

Comment:
This was just renamed - it used to be `test_integration.py` but since it's all about installation and I needed to copy over the integration tests from Jira Cloud, I changed the name.